results/client.go

```diff
@@ -27,12 +24,14 @@ const (
 	PodLogs                   = "podlogs/"
 	MetadataDir               = "meta/"
 
-	ServicesFile      = "Services.json"
-	NodesFile         = "Nodes.json"
-	ServerVersionFile = "serverversion.json"
-	ServerGroupsFile  = "servergroups.json"
+	DefaultServicesFile      = "Services.json"
```

Comment:
Do these need to be public?

Reply:
No, I think the desired use would be through the methods on the Archive struct. Will make them private.